packages/lib/Logger.ts

```diff
@@ -287,6 +291,45 @@ class Logger {
 		}
 	}
 
+	private cleanLogFile(logFileFullPath) {
+		// eslint-disable-next-line promise/prefer-await-to-then
+		return Logger.fsDriver().stat(logFileFullPath).then(({ size }) => {
```

Comment:
Don't use `then()` for new code, simply use `await` instead. As a general rule we should never silence any eslint rule in new code